torch/csrc/jit/runtime/register_prim_ops.cpp

```diff
@@ -2489,9 +2490,46 @@ static const std::vector<OperatorGeneratorArgs> opGenArgs1{
     OperatorGeneratorArgs(
         // TODO return generator object when torchscript supports RNG
         // first-class
-        TORCH_SELECTIVE_SCHEMA("aten::manual_seed(int seed) -> ()"),
+        TORCH_SELECTIVE_SCHEMA("aten::manual_seed.default_(int seed) -> ()"),
         [](Stack& stack) { at::manual_seed(pop(stack).toInt()); },
         aliasAnalysisFromSchema()),
+    OperatorGeneratorArgs(
+        TORCH_SELECTIVE_SCHEMA(
+            "aten::Generator(*, str? device=None, int? seed=None) -> Generator"),
```

Comment:
maybe take a `Device?` instead of a `str?` - unless there's precedent somewhere else to use `str` for device

Reply:
I used `str?` as that was what was being used by `aten::device`

Reply:
I think we should use `Device?` instead - `aten::device` is a special case because its purpose is to convert str -> device; although I'm open to keeping it as `str` if it turns out there's other complications with using `Device`.